lib/core/sync_job_runner.rb

```diff
@@ -81,22 +87,21 @@ def do_sync!
         # occurred as most of the time the main execution thread is interrupted and we miss this Signal/Exception here
         @sync_finished = true
       rescue StandardError => e
-        @status[:error] = e.message
+        @sync_error = e.message
         Utility::ExceptionTracking.log_exception(e)
-        ElasticConnectorActions.update_connector_status(@connector_settings.id, Connectors::ConnectorStatus::ERROR, Utility::Logger.abbreviated_message(e.message))
```

Comment:
Hi @mchernyavskaya , `git blame` told me you added this line, but this is covered by `complete_sync` in the `ensure` block.